### PR TITLE
Finish adding translation support

### PR DESCRIPTION
**Pull request by Sightnado**
Absolutely everything now has translation support.

### CODE REVIEW

**Comment by Nv7-GitHub** [comment]
Holy crap you actually did it

File: eod/categories/get.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (b *Categories) CategoriesCmd(elem string, m types.Msg, rsp types.Rsp) {
 
 	b.base.NewPageSwitcher(types.PageSwitcher{
 		Kind:       types.PageSwitchInv,
-		Title:      fmt.Sprintf("%s Categories (%d)", el.Name, len(out)),
+		Title:      fmt.Sprintf(db.config.LangProperty("ElemCategories"), el.Name, len(out)),
```
Comment (Nv7-GitHub) [inline eod/categories/get.go:60]: Capital C

Comment (Nv7-GitHub) [inline eod/categories/get.go:60]: Capital C

File: eod/categories/get.go [hunk 2]
```diff
@@ -113,7 +113,7 @@ func (b *Categories) DownloadCatCmd(catName string, sort string, postfix bool, m
 	}
 
 	_, err = b.dg.ChannelMessageSendComplex(channel.ID, &discordgo.MessageSend{
-		Content: fmt.Sprintf("Category **%s**:", cat.Name),
+		Content: fmt.Sprintf(db.config.langProperty("NameDownloadedCat"), cat.Name),
```
Comment (Nv7-GitHub) [inline eod/categories/get.go:116]: Capital C

File: eod/categories/get.go [hunk 3]
```diff
@@ -125,5 +125,5 @@ func (b *Categories) DownloadCatCmd(catName string, sort string, postfix bool, m
 	if rsp.Error(err) {
 		return
 	}
-	rsp.Message("Sent category in DMs!")
+	rsp.Message(db.config.LangProperty("CatSentToDMs"))
```
Comment (Nv7-GitHub) [inline eod/categories/get.go:128]: Capital C

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Some compile errors

**Comment by Nv7-GitHub** [comment]
Wait theres actually a lot more missing Capital C stuff, in VSCode go to Search tab, search for `db.config` and replace with `db.Config`

**Comment by Sightnado** [comment]
> 

i don't have vscode installed :(

File: eod/translation/languages/en.json [hunk 4]
```diff
@@ -44,6 +44,68 @@
...
+  
+  "ElemCategories": "%s Categories (%d)",
+  "NameDownloadedCat": "Category **%s**",
+  "CatSentToDMs": "Sent category in DMs!",
+  "InvalidSort": "Invalid sort!",
+  "AllCategories": "All Categories (%d)",
+  
+  "NewAIIdea": "New AI Generated Idea",
+  "FailedAIGenerate": "Failed to generate a valid idea!",
+  "SuggestIdea": "\n 	Suggest it by typing **/suggest**",
```
Comment (Nv7-GitHub) [inline eod/translation/languages/en.json:56]: Make sure to get rid of tab and replace with `\t`

File: eod/translation/languages/en.json [hunk 5]
```diff
@@ -44,6 +44,68 @@
...
   "Stats": "Element Count: **%s**\nCombination Count: **%s**\nMember Count: **%s**\nElements Found: **%s**\nElements Categorized: **%s**",
   "ViewMoreStats": "View More Stats",
+  
+  "ElemCategories": "%s Categories (%d)",
+  "NameDownloadedCat": "Category **%s**",
+  "CatSentToDMs": "Sent category in DMs!",
+  "InvalidSort": "Invalid sort!",
+  "AllCategories": "All Categories (%d)",
+  
+  "NewAIIdea": "New AI Generated Idea",
```
Comment (Nv7-GitHub) [inline eod/translation/languages/en.json:54]: Can you split up the text in this chunk based on command with line breaks? Like code for AI would be in a chunk, then there would be a line break, then code for Info would be in a chunk, etc. Like how it is for the stuff I did

File: eod/translation/schema_config.json [hunk 6]
```diff
@@ -42,5 +42,68 @@
...
+  "ViewMoreStats": "The message in the View More Stats button.",
+  
+  "ElemCats": "The title of the /get categories embed.",
+  "NameDownloadedCat": "Tell the user which category they downloaded.",
+  "CatSentToDMs": "Tell the user that the category was sent to their DMs.",
+  "InvalidSort": "Tell the user the chosen sort is not valid in the current context.",
+  "AllCats": "The title of the embed shown when listing all categories using /cat without a specified category.",
+  
+    
+  "NewAIIdea": "The message in the New AI Generated Idea button.",
```
Comment (Nv7-GitHub) [inline eod/translation/schema_config.json:54]: Same for this thing as for `en.json`, please split up chunks

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Some stuff in the translation file

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Some stuff in the translation file

File: eod/translation/schema_config.json [hunk 7]
```diff
@@ -42,5 +42,72 @@
...
+  
+  "NoResults": "Tells the user that no results were found for the given search parameters.",
+  "ElemSearch": "The title of the Element Search embed, also displaying the number of elements found with the search.",
+  "ElemNameCannotStartWithQuestionMark": "Tells the user that the suggestion cannot start with a question mark.",
+  "ElemNameMaxLength": "Tells the user the maximum number of characters they can have in an element name.",
+  "ElemNameForbiddenChar": "Tells the user the characters in their suggested element name that cannot be in element names.",
+  "NoSuggestElemName": "Tells the user that their suggestion cannot be empty.",
+  "MustSuggestInPlayChannel": "Tells the user that they can only suggest in play channels.",
+  "ComboHasResult": "Tells the user that the combo they are trying to suggest a result for already has a result.",
+  "SuggestedElem": "Tells the user what combo they just suggested.",
```
Comment (Nv7-GitHub) [inline eod/translation/schema_config.json:112]: Can't have , in last one! (just get rid of comma in this line)

File: eod/categories/get.go [hunk 8]
```diff
@@ -113,7 +113,7 @@ func (b *Categories) DownloadCatCmd(catName string, sort string, postfix bool, m
 	}
 
 	_, err = b.dg.ChannelMessageSendComplex(channel.ID, &discordgo.MessageSend{
-		Content: fmt.Sprintf("Category **%s**:", cat.Name),
+		Content: fmt.Sprintf(db.Config.langProperty("NameDownloadedCat"), cat.Name),
```
Comment (Nv7-GitHub) [inline eod/categories/get.go:116]: Capital L!

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
Few bugs, seems pretty good!

File: eod/translation/schema.json [hunk 9]
```diff
@@ -157,6 +157,246 @@
...
+      "description": "Tells the user that they can only suggest in play channels."
+    },
+    "ComboHasResult": {
+      "type": "string",
+      "description": "Tells the user that the combo they are trying to suggest a result for already has a result."
+    },
+    "SuggestedElem": {
+      "type": "string",
+      "description": "Tells the user what combo they just suggested."
+    },
```
Comment (Nv7-GitHub) [inline eod/translation/schema.json:400]: Oh also this needs no comma

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]

**Comment by Sightnado** [comment]
also added an English (UK) translation file for no reason in particular other than to test to see if changing languages actually works

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**